docs/versioned_docs/version-v2.4/02-Features/correlation.md

```diff
@@ -0,0 +1,209 @@
+---
+title: "Correlation"
+layout: default
+---
+
+# Correlation
+
+`CorrelationInfo` provides a common set of correlation levels:
+
+- Transaction Id - ID that relates different requests together into a functional transaction.
+- Operation Id - Unique ID information for a single request.
+- Operation parent Id - ID of the original service that initiated the request.
+
+## Installation
+
+This feature requires to install our NuGet package
+
+```shell
+PM > Install-Package Arcus.Observability.Correlation
+```
+
+## What We Provide
+
+The `Arcus.Observability.Correlation` library provides a way to get access to correlation information across your application.
+What it **DOES NOT** provide is how this correlation information is initially set.
+
+It uses the the Microsoft dependency injection mechanism to register an `ICorrelationInfoAccessor` and `ICorrelationInfoAccessor<>` implementation that is available.
```

Comment:
```suggestion
It uses the Microsoft dependency injection mechanism to register an `ICorrelationInfoAccessor` and `ICorrelationInfoAccessor<>` implementation that is available.
```

Reply:
Again, should be on all, I'll add an issue for this.